Release notes for 2.3 release (#9956)
jupyterlab/jupyterlab: docs/source/developer/notebook.rst

Virtual Notebook
''''''''''''''''

The cells are rendered based on a *Virtual Notebook* logic. A minimun number of cells will be rendered directly 

> (configurable with the `numberCellsToRenderDirectly` setting, then an intersection observer will render any cells

Reviewer: ```suggestion
(configurable with the `numberCellsToRenderDirectly` setting), then an intersection observer will render any cells
```

> being in the observed area (default is the complete notebook panel),and then any other non-rendered cells will

Reviewer: ```suggestion
being in the observed area, which defaults to rendering the complete notebook. Finally any other non-rendered cells will
```

> be rendered when the browser is idle. This ensure that the user has	access to cells as soon as possible.

Reviewer: ```suggestion
be rendered when the browser is idle. This ensures that the user has	access to the complete notebook cells as soon as possible.
```

notebook-specific renderer, since rendering a widget depends on
notebook-specific widget state.

Trimmed Output
''''''''''''''

> A `maxNumberOutputs` setting ensures that large outputs are trimmed down that value. In case of trimmed output,

Reviewer: ```suggestion
The `maxNumberOutputs` setting ensures that large outputs are trimmed down to that value. In case of trimmed output,
```
